src/rag/rag.py

```
@@ -0,0 +1,181 @@
+            try:
+                ebdt = [chunk.data_description for chunk in batch]
+                now_str = datetime.now().strftime("%Y-%m-%d %H:%M")
+                print(f"Embedding: {min(i+batch_count, len(cks))}/{len(cks)} [{now_str}]")
+                embeddings = self.embed(ebdt)
+
+                self.vector_store.add(batch, embeddings)
+                
+                now_str = datetime.now().strftime("%Y-%m-%d %H:%M")
+                print(f'Finish: {min(i+batch_count, len(cks))}/{len(cks)} [{now_str}]')
+                
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)